components/event_dispatcher.rst

```diff
@@ -491,9 +491,9 @@ is dispatched, are passed as arguments to the listener::
     use Symfony\Contracts\EventDispatcher\Event;
     use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
 
-    class Foo
+    class MySubscriber
```

Comment:
why not `MyListener` ? in ref to the above `are passed as arguments to the listener`

Reply:
Both work for me but maybe we should update this sentence too: `are passed as arguments to the listener/subscriber`

Reply:
At the end I changed this for `MyListener` and didn't add "subscriber" to the text description because this section only mentions listeners (there are other sections in this article that talk about subscribers). Cheers!